### ISSUE TITLE
Graindoc: Add `@default` attribute to provide plain language description of the default

### ISSUE THREAD

**Issue by phated** [grain-lang/grain#1991]
I noticed in #1772 that defaults are annoying to document. We should add a new attribute that will be used to describe the default value.

**Comment by alex-snezhko** [comment]
How would this syntax look? I imagine you were thinking something like this?

```
/**
 * @param x: Description of the value
 * @default x: Description of x's default value
 */
```

**Comment by phated** [comment]
I don't think you need to specify the parameter name since we process top-down now (since we started supporting multi line examples)

It'd just be `@default description` on the line after a param
